### DISCUSSION TITLE
Website Issue (?) - Unexpected behavior of website header/logo image

### DISCUSSION THREAD

**Discussion by tonyb983** [remix-run/remix#2517] [Q&A] [answered]
*I apologize in advance if this isn't the correct place to report this, but I didn't see a separate repository for the website itself. I also wasn't sure if this should warrant an "Issue" or a "Discussion".*

I am using Chrome v99.0.4844 on Android 12. 

While on the homepage remix.run, a long tap on the "REMIX" header image does not bring up a context menu like one would expect for a link, it instead takes me into a Google Drive with various versions of the Remix logo.

**Comment by kiliman** [comment, marked as answer]
This is by design. Typically most people would right-click on the logo because they want to save the image. Instead, they redirect you to the Google Drive page that has the logo in various formats. It's kind of an "Easter Egg".

What was the use case you had for long-pressing/right-clicking the logo?
